app/static/css/main.css

```diff
@@ -6,6 +6,10 @@ pre {
   white-space: pre-wrap;
 }
 
```

Comment:
I think you need a dot in here to specify a class (we can’t invent new HTML elements.)  Also, a better name for the class might just be `.monospace`

Reply:
Changed in new commit.